src/drivers/distance_sensor/lightware_laser_i2c/lightware_laser_i2c.cpp

```diff
@@ -127,11 +132,19 @@ class LightwareLaser : public device::I2C, public I2CSPIDriver<LightwareLaser>
 	Type _type{Type::Generic};
 	State _state{State::Configuring};
 	int _consecutive_errors{0};
+
+	DEFINE_PARAMETERS(
+		(ParamInt<px4::params::SF1XX_MODE>) _param_mode
```

Comment:
```suggestion
		(ParamInt<px4::params::SF1XX_MODE>) _param_sf1xx_mode
```
convention is _param_{param name lowercase}